tests/lib/rules/valid-jsdoc.js

```diff
@@ -317,6 +317,21 @@ ruleTester.run("valid-jsdoc", rule, {
             code:
             "/**\n" +
             "* Foo\n" +
+            "* @param {{String:foo}} hi - desc\n" +
```

Comment:
This test is unrelated to my change - just wanted to ensure we were testing that only the value is enforced by the `preferType` option